### PR TITLE
Some Java naming Conventions and better structures replacing loops to to direct frame collection

### PR DESCRIPTION
**Pull request by RANJEETJ06**
- [ ] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized it.
- [ ] All filenames are in PascalCase.
- [x] All functions and variable names follow Java naming conventions.
- [ ] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.

### CODE REVIEW

File: src/main/java/com/thealgorithms/backtracking/MazeRecursion.java [hunk 1]
```diff
@@ -107,14 +105,14 @@ public static boolean setWay(int[][] map, int i, int j) {
...
-                // the current point to 3 and return false, the backtraking will start, it will
+                // the current point to 3 and return false, the backtracking will start, it will
                 // go to the previous step and check for feasible path again
                 map[i][j] = 3;
                 return false;
             }
         } else { // if the map[i][j] != 0 , it will probably be 1,2,3, return false because the
             // ball cannot hit the wall, cannot go to the path that has gone though before,
-            // and cannot head to deadend.
+            // and cannot head to deadened.
```
Comment (siriak) [inline src/main/java/com/thealgorithms/backtracking/MazeRecursion.java:115]: I think this correction is wrong. Please justify the change

File: src/main/java/com/thealgorithms/audiofilters/IIRFilter.java [hunk 2]
```diff
@@ -2,9 +2,9 @@
 
 /**
  * N-Order IIR Filter Assumes inputs are normalized to [-1, 1]
- *
+ * <p>
  * Based on the difference equation from
- * https://en.wikipedia.org/wiki/Infinite_impulse_response
+ * <a href="https://en.wikipedia.org/wiki/Infinite_impulse_response">...</a>
```
Comment (siriak) [inline src/main/java/com/thealgorithms/audiofilters/IIRFilter.java:7]:
```suggestion
 * <a href="https://en.wikipedia.org/wiki/Infinite_impulse_response">here</a>
```

Comment (siriak) [reply src/main/java/com/thealgorithms/audiofilters/IIRFilter.java:7]: Let's add some text to the links

**Review by siriak — CHANGES_REQUESTED** [review]

File: src/main/java/com/thealgorithms/backtracking/NQueens.java [hunk 3]
```diff
@@ -19,10 +19,10 @@
  *
  * Generate all possible arrangement to place N queens on N*N board. Check each
  * board if queens are placed safely. If it is safe, include arrangement in
- * solution set. Otherwise ignore it
- *
+ * solution set. Otherwise, ignore it
+ * <p>
```
Comment (siriak) [inline src/main/java/com/thealgorithms/backtracking/NQueens.java:23]: No need for paragraphs, they complicate reading this in code. Is there a benefit to having them? Would a <br/> work instead?

File: src/main/java/com/thealgorithms/backtracking/MazeRecursion.java (hunk 1 above)
Comment (RANJEETJ06) [reply src/main/java/com/thealgorithms/backtracking/MazeRecursion.java:115]: i will correct it

File: src/main/java/com/thealgorithms/audiofilters/IIRFilter.java (hunk 2 above)
Comment (RANJEETJ06) [reply src/main/java/com/thealgorithms/audiofilters/IIRFilter.java:7]: thanks for the suggestion

File: src/main/java/com/thealgorithms/backtracking/NQueens.java (hunk 3 above)
Comment (RANJEETJ06) [reply src/main/java/com/thealgorithms/backtracking/NQueens.java:23]: i will remove it

**Review by siriak — APPROVED** [review]
Looks good, thanks!

**Comment by RANJEETJ06** [comment]
please assign me as a beginner

**Pull request merged**